pkgs/top-level/ocaml-packages.nix

```diff
@@ -847,6 +847,8 @@ let
 
     frontc = callPackage ../development/ocaml-modules/frontc { };
 
+    ocamlformat-rpc-lib = callPackage ../development/ocaml-modules/ocamlformat-rpc-lib { };
+
```

Comment:
This change belongs to the first commit, isn’t it?

Reply:
Good catch. Fixed.